administration/src/application/components/primitive-inputs/DateForm.tsx

```diff
@@ -0,0 +1,44 @@
+import { TextField } from '@mui/material'
+import { useState } from 'react'
+import { Form } from '../../FormType'
+
+export type DateFormState = { type: 'DateForm'; value: string }
+type ValidatedInput = string
+type Options = void
+type AdditionalProps = { label: string; minWidth?: number }
+const dateForm: Form<DateFormState, Options, ValidatedInput, AdditionalProps> = {
+  initialState: { type: 'DateForm', value: '' },
+  getValidatedInput: ({ value }) => {
+    if (value === '') return { type: 'error', message: 'Feld ist erforderlich.' }
+    const date = Date.parse(value)
+    if (isNaN(date)) {
+      return { type: 'error', message: 'Eingabe ist kein gültiges Datum.' }
+    }
+    return { type: 'valid', value: new Date(date).toString() }
```

Comment:
```suggestion
    return { type: 'valid', value: new Date(date).toLocaleDateString() }
```

https://caniuse.com/date-tolocaledatestring

Reply:
Leaving this for now as it is. Dates need to be supported by the backend first.